### ISSUE TITLE
exporting utility function `getStringWidth` from @vx/text

### ISSUE THREAD

**Issue by sto3psl** [hshoff/vx#300]
`@vx/text` is a great package and has a great utility function `getStringWidth` which I currently use in some charts in the following way (`@vx@0.0.163`): 

```js
import getStringWidth from '@vx/text/build/util/getStringWidth'
```

After installing the `0.0.165-beta.0` this breaks and my charts don't work anymore with the following error: 

```
ReferenceError: babelHelpers is not defined
  at Object.<anonymous> (node_modules/@vx/text/build/util/getStringWidth.js:9:17)
  at Object.<anonymous> (src/utils/utils.js:5:1)
  at Object.<anonymous> (src/utils/utils.test.js:3:14)
```

I'm not quite sure why but I think it has to do with the rollup update and I'm not even supposed to import from `@vx/text/build/util/`.

Maybe this function can get a named export so users of vx can use it with 
```js
import { getStringWidth } from '@vx/text'
```

I can try to come up with a PR but I'm not familiar with Rollup and some directions are really appreciated if this is a feature you want for vx.

**Comment by hshoff** [comment]
Good catch! i need to remove the `external-helpers` babel plugin when we build the `build/` dir of cjs modules.

Will also export the util from `@vx/text`

**Comment by hshoff** [comment]
@sto3psl released in `0.0.165-beta.1` mind giving that version a shot and let me know? you should be able to do either:

```js
import getStringWidth from '@vx/text/build/util/getStringWidth';
// or
import { getStringWidth } from '@vx/text';
``

**Comment by sto3psl** [comment]
@hshoff from a first quick look the charts all work. Will test it tomorrow in our actual app with real data but it looks good. Thanks for the quick update!
